tests/data_access.rs

```diff
@@ -28,7 +28,7 @@ struct Loaded {
 }
 
 const TRACKABLE_MAX: u64 = 3600 * 1000 * 1000;
-const SIGFIG: u32 = 3;
+const SIGFIG: u8 = 3;
```

Comment:
Oh, interesting... I wasn't aware of that at all. Do we check that `SIGFIG > 5` is never used somewhere currently?